qt/python/mantidqtpython/CMakeLists.txt

```diff
@@ -10,7 +10,6 @@ set ( SIP_SRC ${CMAKE_CURRENT_BINARY_DIR}/sip_mantidqt.cpp )
 set ( SIP_SRC_AUTO sipmantidqtpythonpart0.cpp )
 
 # Sip needs to have a dependency on all headers within it
-set ( WIDGETS_SRC_DIR ../../widgets )
```

Comment:
@mantid-roman This line has been deleted but is required by the `SIP_HDRS` macro. It was introduced to avoid having to repeat `../../widgets` everywhere. Does it need to be removed?